wkcuber/api/layer.py

```diff
@@ -607,6 +621,29 @@ def _initialize_mag_from_other_mag(
             compress=compress,
         )
 
+    def set_view_configuration(self, view_configuration: ViewConfiguration) -> None:
+        self.dataset.properties._data_layers[
+            self.name
+        ]._default_view_configuration = vars(view_configuration)
```

Comment:
I am not familiar with the use case of this issue. I have assumed that this refers to the entry `defaultViewConfiguration` in the properties. Is this correct or is there an additional view configuration besides `defaultViewConfiguration`?